easybuild/easyconfigs/v/VSim/VSim-12.2.0.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'PackedBinary'
+
+name = 'VSim'
+
+version = '12.2.0'
+homepage = 'https://www.txcorp.com/vsim'
+description = "VSim is a flexible, multiplatform, multiphysics simulation software tool."
+
+toolchain = {'name': 'system', 'version': ''}
```

Comment:
```suggestion
toolchain = SYSTEM
```

Reply:
Thanks @Miket; I had forgotten about the SYSTEM constant and this change has been made.